contrib/mapping-tester/preparemeshes.py

```diff
@@ -62,6 +66,8 @@ def main(argv):
     partitions = set([int(rank) for pranks in setup["general"]["ranks"].values() for rank in pranks])
 
     for name, file in set(itertools.chain(setup["general"]["meshes"]["A"].items(), setup["general"]["meshes"]["B"].items())):
+        if not os.path.isfile(os.path.expandvars(file)):
+            raise Exception('\033[91m Unable to open file called \"{}\".\033[0m'.format(file))
```

Comment:
This can be shortened a bit.
Not sure what the correct Exception type would be in this instance.

```suggestion
            raise Exception(f'\033[91m Unable to open file called "{file}".\033[0m')
```